content/docs/command-reference/add.md

```diff
@@ -59,34 +68,28 @@ more details.
 To avoid adding files inside a directory accidentally, you can add the
 corresponding [patterns](/doc/user-guide/dvcignore) in a `.dvcignore` file.
 
-By default DVC tries to use reflinks (see
+By default, DVC tries to use reflinks (see
 [File link types](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache)
 to avoid copying any file contents and to optimize DVC-file operations for large
 files. DVC also supports other link types for use on file systems without
 `reflink` support, but they have to be specified manually. Refer to the
 `cache.type` config option in `dvc config cache` for more information.
 
+### Tracking directories
+
 A `dvc add` target can be an individual file or a directory. There are two ways
```

Comment:
I would further deemphasize --recursive. Let's remove `Two ways` reference. Let's describe the default only. And only after that mention that `--recursive` option changes the way DVC deals with dirs. To some extent I would even  say that `--recusrsive` in a way to deal with files, not directories.

Reply:
Done in 13dce7c6.